src/components/files/fileHandler.ts

```diff
@@ -107,7 +115,32 @@ export async function saveFiles(data: any, queryParams: HttpQueryParameters) {
   return filesInfo
 }
 
-export async function registerFileInDB({
+// For registering a file that already exists on disk
```

Comment:
The existing `registerFileInDB` function assumed the files were coming from uploading via endpoint. However, this is not the case any more, so I've changed that function to be just `saveToDB`, which is essentially all it was doing. Now "register" normalises the file data based on limited input and then passes to `saveToDB`.

This made it much easier for calling from "generateTextFile" action

Reply:
But we don't have a use case to save generated text file in db ? 

I haven't reviewd this change, is it necessary ?